src/allotropy/parsers/unchained_labs_lunatic/unchained_labs_lunatic_structure.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any, Optional
+import uuid
+
+import pandas as pd
+
+from allotropy.exceptions import AllotropeConversionError
+from allotropy.parsers.unchained_labs_lunatic.constants import (
+    INCORRECT_WAVELENGHT_COLUMN_FORMAT_ERROR_MSG,
+    NO_DATE_OR_TIME_ERROR_MSG,
+    NO_MEASUREMENT_IN_PLATE_ERROR_MSG,
+    NO_WAVELENGHT_COLUMN_ERROR_MSG,
+    WAVELENGHT_COLUMNS_RE,
+)
+from allotropy.parsers.utils.values import try_float_from_series, try_str_from_series
+
+
+@dataclass(frozen=True)
+class Measurement:
+    measurement_identifier: str
+    wavelenght: float
```

Comment:
typo: should be `wavelength`

Reply:
oops, I just realized I made this typo everywhere!